src/internals/text.js

```diff
@@ -217,8 +215,9 @@ extend(ChartInternal.prototype, {
 		}
 		// show labels regardless of the domain if value is null
 		if (d.value === null && !$$.config.axis_rotated) {
-			if (yPos < box.height) {
-				yPos = box.height;
+			let boxHeight = textElement.getBoundingClientRect().height;
```

Comment:
It's okay for `getXForText()`, but for `getYForText()`, getBoundingClientRect() are called multiple times.
The height value would be gotten from the above if the execution condition is met.

How about defining an inner function, which get the height calling getBoundingClientRect() just once?

```js
	getYForText(points, d, textElement) {
		const $$ = this;
		let yPos;
		let getHeight = () => {
			const height = textElement.getBoundingClientRect().height;

			getHeight = () => height;
			return height;
		};
		
		if ($$.config.axis_rotated) {
			yPos = (points[0][0] + points[2][0] + getHeight() * 0.6) / 2;
		} else {
			...
		}
		
		// show labels regardless of the domain if value is null
		if (d.value === null && !$$.config.axis_rotated) {
			if (yPos < getHeight()) {
				yPos = getHeight();
				
		..
```